config/schema/ultimate_cron.schema.yml

```diff
@@ -135,76 +134,74 @@ ultimate_cron.job.*:
       type: string
       label: 'Callback'
     scheduler:
-      type: mapping
+      type: ultimate_cron.job.*.plugins
       label: 'Scheduler'
-      mapping:
-        id:
-          type: string
-          label: 'Scheduler'
-        configuration:
-          type: mapping
-          label: 'Scheduler configuration'
-          mapping:
-            rules:
-              type: sequence
-              label: Scheduler rules
-              sequence:
-                - type: string
-                  label: Scheduling rule
-            catch_up:
-              type: integer
-              label: 'Timeout (s) after job run'
     launcher:
+      type: ultimate_cron.job.*.plugins
+      label: 'Launcher'
+    logger:
+      type: ultimate_cron.job.*.plugins
+      label: 'Logger'
+
+ultimate_cron.job.*.plugins:
+  type: mapping
+  label: 'Scheduler'
+  mapping:
+    id:
+      type: string
+      label: [type]
+    configuration:
+      type: ultimate_cron.job.*.plugins.settings.[%parent.%key]
```

Comment:
This is where we have problems, when we "hardocde" any of the plugins, it works for the hardcoded plugin. So we want to have the key of the parent plugin mapping, I guess? But the tests are failing...